README.md

```diff
@@ -1,56 +1,60 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Countries
 
-> Description the project.
+> Countries is a small we app which gets south asian countries details from an api and shows its' details like name, flag, lag, long and population.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript
+- React
+- Redux
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://astonishing-piroshki-2b21e0.netlify.app/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Install node.js and npm.
+- Install Git.
+
 ### Setup
 
+- Clone the repository by running the following command in your terminal:
+    git clone https://github.com/HabibPayenda/Countries.git
+    
+- Change your directory by by running the following command in your terminal:
+    cd Countries
+
 ### Install
+- Install the dependencies by running npm i in your terminal
 
 ### Usage
 
+- After all the dependencies are installed open the app by running npm start in your terminal
+
 ### Run tests
 
+To test the application run npm test in your terminal.
+
 ### Deployment
 
 
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
```

Comment:
- [ ] The README contains an "Acknowledgements" section that should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).